### ISSUE TITLE
tesseract missing dependency `libwebpmux.so` from libwebp

### ISSUE THREAD

**Issue by samuellwn** [termux/termux-packages#4849]
**Problem description**
Package 'tesseract' depends on 'libwebpmux.so' which is found in the package 'libwebp', but tesseract doesn't list libwebp as a dependency.

**Steps to reproduce**
```
pkg uninstall libwebp
pkg install tesseract
tesseract --help
```

**Expected behavior**
tesseract should output usage info.

**Actual behavior**
Linker outputs `CANNOT LINK EXECUTABLE "tesseract": library "libwebpmux.so" not found`.

**Additional information**
```
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.org/packages/ stable main
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# root-repo (sources.list.d/root.list)
deb https://dl.bintray.com/grimler/termux-root-packages-24 root stable
Updatable packages:
All packages up to date
Android version:
9
Kernel build information:
Linux localhost 3.18.131-lineageos-g279e209 #1 SMP PREEMPT Wed Jan 15 16:15:54 UTC 2020 aarch64 Android
Device manufacturer:
Google
Device model:
Pixel
```

**Comment by RossComputerGuy** [comment]
Probably a dependency of tesseract needs webp or when it was configured, webp was found and linked.

**Comment by xeffyr** [comment]
Fixed with `leptonica v1.79.0-1`.
